### ISSUE TITLE
Documentation enhancement - wfpm init

### ISSUE THREAD

**Issue by lindaxiang** [icgc-argo/wfpm#55]
Command `wfpm init` is showing the information as:
```
? Project name / GitHub repo name [github-repo]:
? GitHub account [github-account]:
? Project title [Awesome Workflow Packages]:
? Organization or your name [Your Organization Name]:
? Your email [username@email]:
? Open source license:
```
Is it possible to add more guidance about how to populate each item correctly?

**Comment by junjun-zhang** [comment]
@lindaxiang can you elaborate which one(s) needs more explanation?

Maybe **Organization or your name**? This is what's shown as the owner of copyright, probably we should be more explicit on that. **Your email** is actually not used, we can remove that from `wfpm init`

Anything else?

**Comment by junjun-zhang** [comment]
addressed by #58
